usr/share/rear/output/PXE/default/801_copy_to_http.sh

```diff
@@ -0,0 +1,52 @@
+# 801_copy_to_http.sh
+#
+# copy kernel and initrd to HTTP server for Relax-and-Recover
+#
+# This file is part of Relax-and-Recover, licensed under the GNU General
+# Public License. Refer to the included COPYING for full text of license.
+
+# Nothing to do here when PXE_HTTP_UPLOAD_URL is not set
+if [[ -z "$PXE_HTTP_UPLOAD_URL" ]] ; then
+    DebugPrint "PXE_HTTP_UPLOAD_URL not set, skipping copy"
+    return
+fi
+
+# When PXE_HTTP_UPLOAD_URL equals PXE_TFTP_UPLOAD_URL then the files have
+# already been uploaded by 800_copy_to_tftp.sh and we don't need to do
+# anything here.
+if [[ "$PXE_HTTP_UPLOAD_URL" = "$PXE_TFTP_UPLOAD_URL" ]] ; then
+    LogPrint "PXE_HTTP_UPLOAD_URL = PXE_TFTP_UPLOAD_URL, skipping copy to PXE_HTTP_UPLOAD_URL"
```

Comment:
```suggestion
    Debug "PXE_HTTP_UPLOAD_URL = PXE_TFTP_UPLOAD_URL, upload already done by TFTP code in 800_copy_to_tftp.sh"
```

again, I'm not sure we need this whatsoever but happy to keep as debug